projects/plugins/jetpack/modules/shortcodes/smartframe.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * Smartframe.io embed
+ *
+ * Example URL: https://mikael-korpela.smartframe.io/p/mantymetsa_1630927773870/7673dc41a775fb845cc26acf24f1fe4?t=rql1c6dbpv2
+ * Example embed code: <script src="https://embed.smartframe.io/6ae67829d1264ee0ea6071a788940eae.js" data-image-id="mantymetsa_1630927773870" data-width="100%" data-max-width="1412px"></script>
+ *
+ * @package automattic/jetpack
+ */
+
+if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+	add_action( 'init', 'jetpack_smartframe_enable_embeds' );
+} else {
+	jetpack_smartframe_enable_embeds();
+}
+
+/**
+ * Register smartframe as oembed provider. Add filter to reverse iframes to shortcode. Register [smartframe] shortcode.
+ *
+ * @since 9.3.3
+ */
+function jetpack_smartframe_enable_embeds() {
+	// Support their oEmbed Endpoint.
+	wp_oembed_add_provider( '#https?://(.*?)\.smartframe\.(io|net)/.*#i', 'https://oembed.smartframe.io/', true );
+
+	// Allow script to be filtered to short code (so direct copy+paste can be done).
+	add_filter( 'the_content', 'wpcom_shortcodereverse_smartframe' );
```

Comment:
Any reason this is prefixed as `wpcom_` and not `jetpack_`? Since we're starting from scratch, we can be consistent here. 

```suggestion
	add_filter( 'the_content', 'jetpack_shortcodereverse_smartframe' );
```